$NetBSD: patch-dj,v 1.1 2009/07/17 02:53:52 hasso Exp $

Although DragonFly has a POSIX.1e implementation, it's too old at the
moment to be usable here.

--- support/sys-xattr.c.orig	2012-12-05 17:44:30.000000000 +0000
+++ support/sys-xattr.c
@@ -11,7 +11,7 @@
 
 #include <config.h>
 
-#if defined(HAVE_SYS_XATTR_H) || defined(HAVE_ATTR_ATTR_H) || defined(HAVE_SYS_EXTATTR_H)
+#if defined(HAVE_SYS_XATTR_H) || defined(HAVE_ATTR_ATTR_H) || (defined(HAVE_SYS_EXTATTR_H) && !defined(__DragonFly__))
 
 #include <sys/types.h>
 
